src/ServicePulse.Host/vue/public/js/app.constants.js

```diff
@@ -3,5 +3,5 @@ window.defaultConfig = {
     version: '1.2.0',
     service_control_url: 'http://localhost:33333/api/',
     monitoring_urls: ['http://localhost:33633/'],
-    showPendingRetry: false
-};
+    showPendingRetry: false,
```

Comment:
The build is failing because of this change
```suggestion
    showPendingRetry: true
```